pom.xml

```diff
@@ -230,8 +230,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.7</source>
+                    <target>1.7</target>
```

Comment:
But you should bump code dependency to 1.625.3 to indicate the Java 7 requirement if you actually need it, otherwise stay on Java 6 as that was the Java version supported by Jenkins up until 1.612